snapcraft/plugins/maven.py

```diff
@@ -255,7 +255,7 @@ def _create_symlinks(self):
                 "usr",
                 "lib",
                 "jvm",
-                "java-{}-openjdk-*".format(self.options.maven_openjdk_version),
+                "java-{}-openjdk-*".format(self._java_version),
                 "jre",
```

Comment:
Also remove `jre` here, see comment in gradle.py.

Reply:
correct there is no 'jre' directory when using OpenJDK 11, code should be more forgiving

Reply:
just added tests to prove this error, going to fix tomorrow